crates/project/src/search.rs

```diff
@@ -231,6 +231,15 @@ impl SearchQuery {
                 regex, replacement, ..
             } => {
                 if let Some(replacement) = replacement {
+                    let replacement = Regex::new(r"\\\\|\\n|\\t").unwrap().replace_all(
```

Comment:
Let's store that Regex object in something like OnceCell/[OnceLock](https://doc.rust-lang.org/std/sync/struct.OnceLock.html).

Reply:
sure thing. done